dashboard/test/controllers/lti/v1/account_linking_controller_test.rb

```diff
@@ -3,6 +3,7 @@
 class Lti::V1::AccountLinkingControllerTest < ActionController::TestCase
   setup do
     @user = create(:teacher, email: 'test@lti.com')
+    @admin = create :admin, email: 'admin@lti.com'
```

Comment:
Interestingly, `@admin.admin` here is `true` but this test fails, and when this user is looked up in the link_email endpoint, it's nil.